docs/components/modeler/desktop-modeler/connect-to-camunda-cloud.md

```diff
@@ -1,10 +1,10 @@
 ---
 id: connect-to-camunda-cloud
-title: Connect to Camunda Cloud
-description: "Camunda Modeler can communicate directly with Camunda Cloud."
+title: Connect to Camunda Platform 8
```

Comment:
This entire file (`connect-to-camunda-cloud.md`) is based on an old `main` commit. We need to rebase it to latest `main`. Compare with https://docs.camunda.io/docs/components/modeler/desktop-modeler/connect-to-camunda-cloud/

Reply:
Update: I resolved this by cherry picking. See https://github.com/camunda-cloud/camunda-cloud-documentation/pull/724 -> Amara I asked you for review.

Reply:
I'm going to merge in the whole release branch because it looks like some other recent changes were not merged correctly in Modeler. Sorry about this, there is no good time to make sweeping changes like this.